src/autodiff/cppad/minmax.hpp

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright (c) 2020 CNRS INRIA
+//
+
+#ifndef __pinocchio_autodiff_cppad_minmax_hpp__
+#define __pinocchio_autodiff_cppad_minmax_hpp__
+
+#include "pinocchio/utils/static-if.hpp"
+
+#ifdef __CUDACC__
```

Comment:
Do not redefine EIGEN_DEVICE_FUNC. You should use the definition provided by Eigen.